include/libint2/dfbs_generator.h

```diff
@@ -0,0 +1,305 @@
+/*
+ *  Copyright (C) 2004-2024 Edward F. Valeev
+ *
+ *  This file is part of Libint.
+ *
+ *  Libint is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Libint is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with Libint.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef _libint2_src_lib_libint_dfbs_generator_h_
+#define _libint2_src_lib_libint_dfbs_generator_h_
+
+#include <libint2.h>
+#include <libint2/atom.h>
+#include <libint2/basis.h>
+#include <libint2/boys.h>
+#include <libint2/pivoted_cholesky.h>
+#include <math.h>
+
+#include <Eigen/Dense>
+#include <Eigen/Eigenvalues>
+#include <algorithm>
+
+namespace libint2 {
+
+namespace detail {
+
+/// Computes uncontracted shells from a vector of shells
```

Comment:
`Computes` is usually redundant ... every function computes its result by definition. Better: "decontracts a set of shells" or "uncontracts a set of shells"